htmltools/_core.py

```diff
@@ -1520,8 +1524,8 @@ def copy_to(self, path: str, include_version: bool = True) -> None:
 
         # Collect all the source files
         if self.all_files:
-            src_files = list(Path(paths["source"]).glob("*"))
-            src_files = [str(x) for x in src_files]
+            path_src = Path(paths["source"])
+            src_files = [str(x.relative_to(path_src)) for x in path_src.glob("*")]
```

Comment:
Was this `relative_to` needed before? (And was that a bug?)

Reply:
:facepalm: Didn't mean to check that in. Although I think the new code is better; the old code put absolute paths into the html dependency.

Reply:
I should at least add a test though

Reply:
Sorry, I misremembered. The old code was busted. If you used `all_files=True` and then called `dep.copy_to()`, you'd get an error. The code would come up with a set of absolute paths for the source, and then end up trying to copy the absolute path onto itself, which would cause an error. The new code actually works, just added a unit test.